src/Nox.Reference.Data.Common/Extensions/MapperExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Nox.Reference.Common;
+
+namespace Nox.Reference.Data.Common;
+
+public static class MapperExtensions
+{
```

Comment:
Should we add "ToEntity" extension methods too?

Reply:
I don't think this is good idea. 
1) Since ToEntity() is extension method it will be applied for any object in a solution 
2) We need dto to entity converting only when we write data internaly